src/modules/navigator/navigator_main.cpp

```diff
@@ -1030,6 +1068,36 @@ void Navigator::check_traffic()
 							publish_vehicle_cmd(&vcmd);
 							break;
 						}
+
+					case 3: {
+							/* Land Mode */
+							mavlink_log_critical(&_mavlink_log_pub, "TRAFFIC: %s Landing! dst %d, hdg % d",
+									     tr.flags & transponder_report_s::PX4_ADSB_FLAGS_VALID_CALLSIGN ? tr.callsign : uas_id,
+									     traffic_seperation,
+									     traffic_direction);
+
+							// ask the commander to land
+							vehicle_command_s vcmd = {};
+							vcmd.command = vehicle_command_s::VEHICLE_CMD_NAV_LAND;
+							publish_vehicle_cmd(&vcmd);
```

Comment:
This is okay as a test, but if we want it to exist long term it should really be tied into the state machine a little more directly.